src/components/Uploader/Form.tsx

```diff
@@ -117,7 +117,7 @@ export default class Form extends React.Component<ILayoutProps, ILayoutState> {
 
     public onSave() {
         if (!this.state.savedOnce) {
-            this.state.savedOnce = true;
+            this.setState({savedOnce: true});
             this.shouldComponentUpdate(this.state);
```

Comment:
Non `this.shouldComponentUpdate(this.state);`! De toute façon j'ai mis pas mal tous les packages à jour et ça va tout changer cette partie donc vas-y. La fonction `shouldComponentUpdate`servait à ''updater'' le _state_ du module manuellement...